main.py

```diff
@@ -11,6 +12,11 @@
 my_user_id = getenv('MY_USER_ID')
 tweet_id = getenv('TWEET_ID')
 
+parser = argparse.ArgumentParser()
+parser.add_argument("-s", "--skip_following", help="Skip accounts that you have followed")
+parser.add_argument("-i", "--ignore", nargs='+', type=str, help="ignore and skip block this usernames")
```

Comment:
For consistency and better grammar, I suggest `"Skip blocking this username"` as the help text.